dask/dataframe/tests/test_categorical.py

```diff
@@ -78,3 +86,36 @@ def test_categories():
 
     df3 = dd.categorical._categorize(categories, df2)
     tm.assert_frame_equal(df, df3)
+
+
+class TestCategoricalAccessor:
+
+    @pytest.mark.parametrize('prop, compare', [
+        ('categories', tm.assert_index_equal),
+        ('ordered', eq),
+        ('codes', eq),
+    ])
+    def test_properties(self, cat_series, prop, compare):
+        a = dd.from_pandas(cat_series, npartitions=2)
+        expected = getattr(cat_series.cat, prop)
+        result = getattr(a.cat, prop)
+        compare(result, expected)
+
+    @pytest.mark.parametrize('method, kwargs', [
+        ('add_categories', dict(new_categories=['d', 'e'])),
+        ('as_ordered', {}),
+        ('as_unordered', {}),
+        ('as_ordered', {}),
+        ('remove_categories', dict(removals=['a'])),
+        ('remove_unused_categories', {}),
+        ('rename_categories', dict(new_categories=['d', 'e', 'f'])),
+        ('reorder_categories', dict(new_categories=['a', 'b', 'c'])),
+        ('set_categories', dict(new_categories=['a', 'e', 'b'])),
+    ])
+    def test_callable(self, cat_series, method, kwargs):
```

Comment:
I'm fine with it personally.  Covering the Pandas API comprehensively is hard.  Using parametrize seems like a good match.